src/components/WorkOrders/MobileWorkingWorkOrdersView/MobileWorkingWorkOrdersView.js

```diff
@@ -8,16 +12,34 @@ import WarningInfoBox from '../../Template/WarningInfoBox'
 import Meta from '../../Meta'
 import { WorkOrder } from '../../../models/workOrder'
 import { MobileWorkingWorkOrderListItems } from './MobileWorkingWorkOrderListItems'
+import TabsVersionTwo from '../../TabsVersionTwo/Index'
 
 const SIXTY_SECONDS = 60 * 1000
+let toggleStatus
```

Comment:
Why have you used let here? Is there a reason you cant store the value in useState?

Reply:
I thought that because the state won't change without a developer changing it then there wasn't any point. For the status to change I have to change it in the api file, it won't change from user interaction.

Reply:
yeah, but you should store the variable within the component unless there is a good reason to. Secondly, the state does change, because you need to wait for the request